src/api/Main/BuisnessLayer/Shrooms.Domain/Services/Kudos/IKudosService.cs

```diff
@@ -49,12 +50,24 @@ public interface IKudosService
         /// <param name="points">requested points</param>
         void AddKudosLog(AddKudosLogDTO kudosDto, decimal points);
 
+        /// <summary>
+        /// Must not call SaveChanges inside.

```

Comment:
These statements are not looking very promising..
1. What is "inside" is implementation details. Interface is not implementation. So comments doesn't look relevant here in the first place.
2. On the other hand - if such statements are needed, it shows that implemented pattern is confusing and can introduce breaking changes (in case some one misses the comment).